### ISSUE TITLE
Discord - a group chat incoming message sound

### ISSUE THREAD

**Issue by RozekPoland** [miranda-ng/miranda-ng#1453]
I use sound samples for incoming messages in Miranda. I have been using them for years. When I use Discord plugin I hear the sound all the time because of some group chats that are going on in some guilds(channels) that I am subscribed to (in Discord). I don't even follow them (I have them all hidden at startup). Because of that I have had to disable the incoming group chat message sound.

- **Would it be possible to make a separate Discord incoming group chat message sound entry**?
- **Or maybe it would be possible to trigger/play the sound only when the Discord group chat is open**?

![image](https://user-images.githubusercontent.com/37516069/42169528-49443750-7e14-11e8-9621-feb9b9e5629a.png)

**Comment by Goraf** [comment]
There is a plugin called xSoundNotify. It has option to ignore sounds for specific contact. Could you try it? Maybe it could work as a separate solution

**Comment by RozekPoland** [comment]
Yes, there is and I tested it. Its only useful feature is ignoring all sounds from Discord contacts and guilds(channels). I have just disabled the group chat incoming message sound as can be seen in the picture above which practically does the same without extra plugin ;)

**Comment by georgehazan** [comment]
this enhancement is scheduled, to ignore all Discord channels & use only manually added

**Comment by RozekPoland** [comment]
Hello :)
Has there been any progress on the issue?
